pkg/client/README.md

```diff
@@ -0,0 +1,132 @@
+# William
+
+This library aims to prevent writing to much code using william on your application.
+
+## Creating a service
+```go
+wi := william.New("URL OF WILLIAM", "ServiceName")
+```
+You can set the HTTP Client that will made the request also the tracing if anyone. (if not set nil)
+```go
+wi.SetClient(
+    &http.Client{
+        Timeout: time.Second * 2,
+    },
+    nil,
+)
+```
+
+If your service will use any internal function of william like a service, Ej: adding permission directly from your service and not from the Hub you need to set a KeyPair to auth.
+
+```go
+wi.SetKey(
+    "ServiceKeyID",
+    "ServiceKeySecret",
+)
+```
+
+By default William is created with a bypass, you can enable with:
+```go
+wi.Enable()
+```
+
+## Permissions Handler
+You want to validate if a user have a valid permission for a specific Handler. If your application don't care about resource you can use william.Generate("RL", "Action") that will wrap this for you.
+```go
+mux := http.NewServeMux()
+mux.HandleFunc("/action",
+    wi.HandlerFunc(wi.Generate("RL", "Action"),
```

Comment:
What the `Generate` method does?

Reply:
Generate create a static permission with the name of the service. Ex: ServicesStatus::RL::Action::*
Also, this function is variadic, you can add more strings at the final:
wi.HandlerFunc(wi.Generate("RL", "Action", "Resource1", "Resource2")
will generate: 
ServicesStatus::RL::Action::Resource1::Resource2